### DISCUSSION TITLE
Small script to detect commands you use the most to contribute them

### DISCUSSION THREAD

**Discussion by iTrooz** [tldr-pages/tldr#12293] [Show and tell]
https://gist.github.com/iTrooz/8a164ea2821fb8b1f017c207ee3627dc

Running this script in this repository's root will output you the commands you use the most that are not in tldr yet

**Comment by kbdharun** [comment]
Cool script, thanks for creating it. If possible, can you add it to the Wiki at <https://github.com/tldr-pages/tldr/wiki/Useful-scripts-and-programs>?

**Reply by iTrooz** [reply]
Sure ! I see there is already a script that does this (although with less features AFAIK) in the wiki. Should I create a new section, or add my script in the same section ?

**Reply by kbdharun** [reply]
IG you could add it under the same section.

**Reply by iTrooz** [reply]
Ok !

**Reply by sbrl** [reply]
Multiple different choices are fine :-)

**Comment by ZeyuanGuo** [comment]
if i don't use zsh, it will report error when trying to open .zsh_history
maybe should add a if file exists check before open history ?
btw, now this scripts need to put into root of the repo, maybe can use the interface to get commands that has in tldr,like :`https://api.github.com/repos/tldr-pages/tldr/contents/pages.zh/windows`(format is `https://api.github.com/repos/{owner}/{repo}/contents/{path}`).

**Reply by iTrooz** [reply]
> if i don't use zsh, it will report error when trying to open .zsh_history

Whoooops, fixed it

> btw, now this scripts need to put into root of the repo, maybe can use the interface to get commands that has in tldr,like :https://api.github.com/repos/tldr-pages/tldr/contents/pages.zh/windows(format is https://api.github.com/repos/{owner}/{repo}/contents/{path}).

Could be nice, but I don't feel like doing it rn. If you want to, I can create a repository for the script and you can submit a PR
